content/zh/docs/tutorials/how-to-profiler-apiserver.md

```diff
@@ -12,9 +12,9 @@ description: >
 
 在教程开始前，您需要确定 kubernetes 集群中已经正确安装 KubeDiag，并且确认有以下资源
 
-* ServiceAccount：[apiserver-profiler-sa](../../config/rbac/apiserver_profiler_viewer_serviceaccount.yaml)
-* ClusterRole：[apiserver-profiler-role](../../config/rbac/apiserver_profiler_viewer_clusterrole.yaml)
-* ClusterRoleBinding：[apiserver-profiler-rolebinding](../../config/rbac/apiserver_profiler_viewer_binding.yaml)
+* ServiceAccount：[apiserver-profiler-sa](https://github.com/kubediag/kubediag/blob/master/config/rbac/apiserver_profiler_viewer_serviceaccount.yaml)
+* ClusterRole：[apiserver-profiler-role](https://github.com/kubediag/kubediag/blob/master/config/rbac/apiserver_profiler_viewer_clusterrole.yaml)
+* ClusterRoleBinding：[apiserver-profiler-rolebinding](https://github.com/kubediag/kubediag/blob/master/config/rbac/apiserver_profiler_viewer_binding.yaml)
```

Comment:
These could be walked through by running a couple of commands:

```bash
kubectl get serviceaccount -n kubediag apiserver-profiler-sa        
NAME                    SECRETS   AGE
apiserver-profiler-sa   1         5d6h
```

Reply:
Copied.